packages/flutter/lib/src/cupertino/bottom_tab_bar.dart

```diff
@@ -213,7 +213,7 @@ class CupertinoTabBar extends StatelessWidget implements PreferredSizeWidget {
           Expanded(
             child: Semantics(
               selected: active,
-              // TODO(xster): This needs localization support. https://github.com/flutter/flutter/issues/13452
+              // Change what?
```

Comment:
@xster can you advise what change is needed here now that the issue is resolved? I am thinking maybe @shihaohong would have some thoughts?

Reply:
oh thanks for going through this again and cleaning up!
This one in particular would need a new entry in cupertino_en.arb for this announcement. Once it goes through the translation console, it should read through CupertinoLocalization rather than having a string here.

Reply:
Awesome! Thanks!

Reply:
See also https://github.com/flutter/flutter/pull/13714/files where this was done for Material.

Reply:
Basically what @xster said about the `cupertino_en.arb` file. After that, you'll need to run [gen_missing_localizations.dart](https://github.com/flutter/flutter/blob/master/dev/tools/localization/bin/gen_missing_localizations.dart), which adds a placeholder 'TBD' for all the other locales' arb files. At this point, you can basically read it off of `CupertinoLocalization`, like in the PR @goderbauer shared.

Once the translation console has the translated messages, we run a script to refresh localizations periodically, which will then update the message for the non-English locales. This can come with a follow-up PR since we don't know how long the translations will take.

Reply:
I've split this out for its own PR: #55336